index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="styles.css" />
+		<link
+			href="https://fonts.googleapis.com/css?family=Poppins"
+			rel="stylesheet"
+		/>
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@700&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+			rel="stylesheet"
+		/>
+		<title>My Portfolio</title>
+	</head>
+	<body>
+		<header>
+			<a href="index.html" class="primary nav-header">My Logo</a>
+			<nav>
+				<img class="nav-img" src="./images/menu.svg" alt="menu" />
+			</nav>
+		</header>
+
+		<section class="personal-info">
+			<h1>
+				I’m Mai
+				<br />
+				Glad to see you!
+			</h1>
+			<p class="intro">
+				I’m a software developer! I can help you build a product , feature or
+				website Look through some of my work and experience! If you like what
+				you see and have a project you need coded, don’t hestiate to contact me.
+			</p>
+			<p class="primary connect">LET’S CONNECT</p>
+			<ul class="social-links">
+				<li>
+					<a class="link" href="#">
+						<img src="./images/twitter.svg" alt="twitter" />
+					</a>
+				</li>
+				<li>
+					<a class="link" href="#">
+						<img src="./images/linkedin.svg" alt="linkedin" />
+					</a>
+				</li>
+				<li>
+					<a class="link" href="#">
+						<img src="./images/medium.svg" alt="medium" />
+					</a>
+				</li>
+				<li>
+					<a class="link" href="#">
+						<img src="./images/github.svg" alt="github" />
+					</a>
+				</li>
+				<li>
+					<a class="link" href="#">
+						<img src="./images/angellist.svg" alt="angellist" />
+					</a>
+				</li>
+			</ul>
+		</section>
```

Comment:
## OPTIONAL

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_386](https://user-images.githubusercontent.com/69638013/185082485-6299fa7a-c311-4daa-a591-e7f94ef23da0.png)  |  ![Screenshot_387](https://user-images.githubusercontent.com/69638013/185082606-b8974f6e-aba0-4063-8d9c-43681a5f42d1.png)


- [ ] Kindly add the` border-bottom-left-radius `  this will make your design look exactly like the figma design